pkg/lib/aws/errors.go

```diff
@@ -128,10 +128,11 @@ func ErrorBucketNotFound(bucket string) error {
 	})
 }
 
-func ErrorInstanceTypeLimitIsZero(instanceType string, region string) error {
+func ErrorInsufficientInstanceQuota(instanceType string, lifecycle string, region string, requiredInstances int64, vCPUPerInstance int64, vCPUQuota int64, quotaCode string) error {
+	url := fmt.Sprintf("https://%s.console.aws.amazon.com/servicequotas/home?region=%s#!/services/ec2/quotas/%s", region, region, quotaCode)
 	return errors.WithStack(&errors.Error{
-		Kind:    ErrInstanceTypeLimitIsZero,
-		Message: fmt.Sprintf(`you don't have access to %s instances in %s; please request access in the appropriate region (https://console.aws.amazon.com/support/cases#/create?issueType=service-limit-increase&limitType=ec2-instances). If you submitted a request and it was recently approved, please allow ~30 minutes for AWS to reflect this change."`, instanceType, region),
+		Kind:    ErrInsufficientInstanceQuota,
+		Message: fmt.Sprintf("your cluster may require up to %d %s %s instances, but your AWS quota for %s %s instances in %s is only %d vCPU (there are %d vCPUs per %s instance); please reduce the maximum number of %s %s instances your cluster may use (e.g. by changing max_instances and/or spot_config if applicable), or request a quota increase to at least %d vCPU here: %s (if your request was recently approved, please allow ~30 minutes for AWS to reflect this change)", requiredInstances, lifecycle, instanceType, lifecycle, instanceType, region, vCPUQuota, vCPUPerInstance, instanceType, lifecycle, instanceType, requiredInstances*vCPUPerInstance, url),
```

Comment:
note to reviewer: this reads like this:

error: dev/config/cluster.yaml: instance_type: your cluster may require up to 99999 spot t3a.small instances, but your AWS quota for spot t3a.small instances in us-west-2 is only 27000 vCPU (there are 2 vCPUs per t3a.small instance); please reduce the maximum number of spot t3a.small instances your cluster may use (e.g. by changing max_instances and/or spot_config if applicable), or request a quota increase to at least 199998 vCPU here: https://us-west-2.console.aws.amazon.com/servicequotas/home?region=us-west-2#!/services/ec2/quotas/L-34B43A08 (if your request was recently approved, please allow ~30 minutes for AWS to reflect this change)